middleware/grpc/baggage_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2021 The OpenZipkin Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package grpc_test
+
+import (
+	"context"
+	"net"
+	"testing"
+
+	"github.com/openzipkin/zipkin-go/idgenerator"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/protobuf/types/known/emptypb"
+
+	"github.com/openzipkin/zipkin-go"
+	zipkingrpc "github.com/openzipkin/zipkin-go/middleware/grpc"
+	"github.com/openzipkin/zipkin-go/propagation/baggage"
+	service "github.com/openzipkin/zipkin-go/proto/testing"
+)
+
+const ReqID = "x-request-id"
```

Comment:
Should this be exported? I don't see this being used outside the package.

Reply:
no I'm unexporting it... since these live in *_test packages there was no real harm to it, but better still to adjust.